recipes/openssl/1.x.x/conanfile.py

```diff
@@ -603,6 +606,9 @@ def build(self):
                 env_vars["CROSS_TOP"] = os.path.dirname(os.path.dirname(xcrun.sdk_path))
             with tools.environment_append(env_vars):
                 if self._full_version >= "1.1.0":
+                    if self.settings.os == "tvOS":
+                        tools.patch(patch_file=os.path.join("patches", "1.1.1-tvos.patch"),
+                                    base_path=self._source_subfolder)
```

Comment:
I don't like the conditional patching.
Can the patch be rewritten so it only changes behavior for WatchOS?
Maybe by checking for the existence for a certain preprocessor variable that can be set in the `conanfile.py`?

Reply:
Considering this is the patch that is being used as the fix for this in the [official OpenSSL repository](https://github.com/openssl/openssl/issues/7607), I don't think that makes sense. Furthermore, since even if you pass `-DNO_FORK`, [it gets undefined in `apps/ocsp.c`](https://github.com/openssl/openssl/issues/7607#issuecomment-596900929), that is a bug in OpenSSL itself that is _not_ specific to tvOS, so I would argue that it shouldn't be made specific to tvOS.

Reply:
Ok. Good point.